qiskit/circuit/register.py

```diff
@@ -152,4 +153,4 @@ def __eq__(self, other):
 
     def __hash__(self):
         """Make object hashable, based on the name and size to hash."""
-        return self._hash
+        return self._hash
```

Comment:
```suggestion
        return self._hash

```
The linter is warning about a missing newline character at the end of the file. I think this should fix it.